.coreos/desoto.service

```diff
@@ -9,23 +9,13 @@ Restart=always
 TimeoutStartSec=10 min
 User=core
 KillMode=none
-
 EnvironmentFile=/etc/environment
-
-ExecStartPre=/usr/bin/env bash -c "etcdctl get /environments/%p > '/tmp/%p.env'"
-
+ExecStartPre=/usr/bin/env bash -c "etcdctl --no-sync get /environments/%p > '/tmp/%p.env' && echo \"ETCD_HOSTS=http://`ifconfig docker0 | awk '/\<inet\>/ { print $2}'`:4001\" >> '/tmp/%p.env'"
 ExecStartPre=-/usr/bin/env docker stop "%p"
 ExecStartPre=-/usr/bin/env docker rm "%p"
 ExecStartPre=/usr/bin/env docker pull christianbladescb/desoto
-
-ExecStart=/usr/bin/env docker run \
-  --name='%p' \
-  --env-file="/tmp/%p.env" \
-  -e HOST=${COREOS_PRIVATE_IPV4} \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  christianbladescb/desoto
-
+ExecStart=/usr/bin/env docker run    --name='%p'    --env-file="/tmp/%p.env" -e HOST=${COREOS_PRIVATE_IPV4}    -v /var/run/docker.sock:/var/run/docker.sock    christianbladescb/desoto
```

Comment:
I'm not sure why the formatting of this was changed (unless I'm missing a content change). I personally favor the original formatting for readability.